question/bank/managecategories/tests/behat/question_categories_idnumber.feature

```diff
@@ -24,31 +24,36 @@ Feature: A teacher can put questions with idnumbers in categories with idnumbers
       | Course       | C1        | Top              | top            |          |
       | Course       | C1        | top              | Used category  | c1used   |
     And I navigate to "Question bank" in current page administration
-    And I select "Categories" from the "questionbankactionselect" singleselect
+    And I click on "jump" "select"
+    And I click on "Categories" "option"
+    And I press "Add category"
     And I set the following fields to these values:
-      | Name            | Sub used category |
-      | Parent category | Used category     |
+      | Name            | New cat |
+      | Parent category | Top for Course 1  |
       | Category info   | Created as a test |
       | ID number       | c1used            |
-    And I press "Add category"
+    And I click on "//div[@class='modal-footer']/button[@data-action='save']" "xpath_element"
     # Standard warning.
     Then I should see "This ID number is already in use"
     # Correction to a unique idnumber for the context.
     And I set the field "ID number" to "c1unused"
-    And I press "Add category"
-    Then I should see "Sub used category"
+    And I click on "//div[@class='modal-footer']/button[@data-action='save']" "xpath_element"
```

Comment:
Can you try anything like the following instead?
And I click on "Close" "button" in the ".modal-dialog" "css_element"

Reply:
Changed to `And I click on "Add category" "button" in the ".modal-footer" "css_element"`